src/quest_manager/static/quest_manager/js/bootstrap-table-accordion.js

```diff
@@ -1,3 +1,27 @@
+/**
+ * The dateSorter function is used to sort table cells by their date value in a Bootstrap table.
+ *
+ * It works by extracting timestamps embedded as data attributes ('data-timestamp') in the HTML structure of the table cells.
+ * These timestamps are then used for comparison to determine the order of cells.
+ * If a cell does not contain a timestamp, a default value of 0 is used.
+
+ * @param {string} a - The inner HTML of the first table cell to compare. This should include a 'data-timestamp' attribute.
+ * @param {string} b - The inner HTML of the second table cell to compare. This should also include a 'data-timestamp' attribute.
+ *
+ * @returns {number} -1, 0, or 1, depending on whether the first cell's date is less than, equal to, or greater than the second.
+ */
+function dateSorter(a, b) {
+
+  // Get the timestamp from the end of the string. This will break if the HTML structure changes.
+  var timestampA = a.includes('data-timestamp') ? parseFloat(a.split('data-timestamp="')[1].split('"')[0]) : 0;
+  var timestampB = b.includes('data-timestamp') ? parseFloat(b.split('data-timestamp="')[1].split('"')[0]) : 0;
+
+  // Compare the timestamp values.
+  if (timestampA < timestampB) return -1;
+  if (timestampA > timestampB) return 1;
+  return 0;
+}
+
```

Comment:
Uneccessary.  Just use a column with `data-visible='false'`with a numeric value for the datetime (see last_sub_datetime column in profile_list.html) and sort based on that column via `data-sort-name='name_of_other_column'`

This info was in the original issue.

Reply:
This was changed as per the description in the original issue. I also moved it to a new pull request:
https://github.com/bytedeck/bytedeck/pull/1420